src/openforms/registrations/contrib/objects_api/config.py

```diff
@@ -183,54 +183,68 @@ def get_fields(self):
 
         # If in an import context, we don't want to error on missing groups.
         # Instead, we will try to set one if possible in the `validate` method.
+        # We also change the `objecttype` field to be any string, as it could be an URL.
```

Comment:
Not ideal, but I can't handle this in `to_internal_value` anymore as I need the `objecttype` to be an URL so that I can select the correct group based on it (and `to_internal_value` runs before `validate`)

Reply:
it's documented, that's the important part :)